frontend/src/views/supp-allowances/SupplementarySubmitView.vue

```diff
@@ -236,6 +247,9 @@ export default {
       try {
         this.loading = true
         this.$emit('process', true)
+        if (!this.currentOrg) {
```

Comment:
If the provider updates their inclusion policy to Yes but they don't refresh the page, the inclusion policy in Org store will not be updated (e.g.: still set to False) --> I think we should always reload getOrganizationInfo().

Also, I think we should call getOrganizationInfo() in SupplementaryAllowanceView instead of calling it twice in SupplementaryFormView and SupplementarySubmitView.